### PR TITLE
UAR-311 Tell us about the Individual BO

### PR DESCRIPTION
**Pull request by cdrainCH**
### JIRA link

https://companieshouse.atlassian.net/jira/software/c/projects/UAR/boards/531?modal=detail&selectedIssue=UAR-311&sprint=2942

### Change description

Update version of the "Tell about the individual" page

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: test/controllers/update/update.trusts.individuals.or.entities.involved.controller.spec.ts
Comment (joshuahigginsonCH) [inline test/controllers/update/update.trusts.individuals.or.entities.involved.controller.spec.ts:1]: Why did you remove the active feature return?

Comment (cdrainCH) [reply test/controllers/update/update.trusts.individuals.or.entities.involved.controller.spec.ts:1]: It's already present in the test so no need to repeat it, just noticed it when I was writing the other tests

**Review by joshuahigginsonCH — APPROVED** [review]

File: src/controllers/trust.involved.controller.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { NextFunction, Request, Response } from 'express';
-import { getTrustInvolvedPage, postTrustInvolvedPage } from "../utils/trust/trust.involved";
+import { getTrustInvolvedPage, postTrustInvolvedPage } from "../utils/trust.involved";
```
Comment (stephanie-ch) [inline src/controllers/trust.involved.controller.ts:2]: How come a number of files have been brought up a directory, out of the trusts directory?

File: views/update/update-trust-tell-us-about-individual-bo.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{% extends "update-layout.html" %}
+
+{% set title = "Tell us about an individual (Trusts)" %}
```
Comment (stephanie-ch) [inline views/update/update-trust-tell-us-about-individual-bo.html:3]: Is this title correct?

File: src/routes/index.ts [hunk 3]
```diff
@@ -518,6 +519,27 @@ router.route(config.UPDATE_TRUSTS_ASSOCIATED_WITH_THE_OVERSEAS_ENTITY_URL)
...
+    navigation.hasUpdatePresenter,
+    navigation.hasTrustDataUpdate
+  )
+  .get(updateTrustIndividualBeneficialOwner.get)
+  .post(...validator.trustIndividualBeneficialOwner, updateTrustIndividualBeneficialOwner.post);
+
+router
+  .route(config.UPDATE_TRUSTS_INDIVIDUALS_OR_ENTITIES_INVOLVED_URL + config.TRUST_ID + config.UPDATE_TRUST_INDIVIDUAL_BENEFICIAL_OWNER_URL + config.TRUSTEE_ID + '?')
+  .all(
+    isFeatureEnabled(config.FEATURE_FLAG_ENABLE_TRUSTS_WEB),
```
Comment (stephanie-ch) [inline src/routes/index.ts:536]: Wrong flag?

File: src/controllers/trust.involved.controller.ts (hunk 1 above)
Comment (cdrainCH) [reply src/controllers/trust.involved.controller.ts:2]: I initially had the trust.involved file in the utils/trust folder, but after speaking with Cory took it out again and put it with the rest of the trusts files

File: src/routes/index.ts (hunk 3 above)
Comment (cdrainCH) [reply src/routes/index.ts:536]: Ah yes it is, good spot!

File: views/update/update-trust-tell-us-about-individual-bo.html (hunk 2 above)
Comment (cdrainCH) [reply views/update/update-trust-tell-us-about-individual-bo.html]: Fixed

File: src/routes/index.ts (hunk 3 above)
Comment (cdrainCH) [reply src/routes/index.ts]: Fixed (route removed, only needed one)

File: src/controllers/trust.involved.controller.ts (hunk 1 above)
Comment (stephanie-ch) [reply src/controllers/trust.involved.controller.ts:2]: What's the reason? There are a number of files still in the /trust directory

Comment (cdrainCH) [reply src/controllers/trust.involved.controller.ts:2]: The trusts folder is specifically for holding trust related business logic thats used by the controllers, so we're separating it from our files. A separate folder for the trusts shared controller would be ideal though

**Review by conuirchgov — CHANGES_REQUESTED** [review]

File: views/update/update-trust-tell-us-about-individual-bo.html (hunk 2 above)
Comment (cdrainCH) [reply views/update/update-trust-tell-us-about-individual-bo.html]: Ah right, that's how it was written when I started and is present on main. To remain consistent I'll put the string back in

Comment (stephanie-ch) [reply views/update/update-trust-tell-us-about-individual-bo.html]: Can you please ask Matthew what the content should be? Then the pattern for update is not to use page params but it have it in the HTML.

**Review by stephanie-ch — APPROVED** [review]

File: src/controllers/trust.individual.beneficial.owner.controller.ts [hunk 4]
```diff
@@ -1,133 +1,10 @@
...
-import { mapIndividualTrusteeToSession, mapIndividualTrusteeFromSessionToPage } from '../utils/trust/individual.trustee.mapper';
-import { safeRedirect } from '../utils/http.ext';
-import { FormattedValidationErrors, formatValidationError } from '../middleware/validation.middleware';
-import { validationResult } from 'express-validator';
-import { Session } from '@companieshouse/node-session-handler';
-import { saveAndContinue } from '../utils/save.and.continue';
+import { getTrustIndividualBo, postTrustIndividualBo } from "../utils/trust.individual.beneficial.owner";
 
-const INDIVIDUAL_BO_TEXTS = {
-  title: 'Tell us about the individual',
```
Comment (stephanie-ch) [inline src/controllers/trust.individual.beneficial.owner.controller.ts:18]: It looks like the title didn't have "(trusts)"? So I'm confused now haha :D

File: views/update/update-trusts-tell-us-about-individual-bo.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{% extends "update-layout.html" %}
+
+{% set title = "Tell us about an individual (Trusts)" %}
```
Comment (stephanie-ch) [inline views/update/update-trusts-tell-us-about-individual-bo.html:3]: If the steer from Matthew is keep it the same as registration, we should drop "(trusts)"

Comment (stephanie-ch) [reply views/update/update-trusts-tell-us-about-individual-bo.html:3]: I've approved but please fix this before merging

**Review by conuirchgov — APPROVED** [review]
